test/pipelines/test_eval.py

```diff
@@ -2069,7 +2068,8 @@ def test_load_legacy_evaluation_result(tmp_path):
     assert "content" not in eval_result["legacy"]
 
 
-def test_load_evaluation_result_w_none_values(tmp_path):
+@pytest.mark.unit
+def test_load_evaluation_result(tmp_path):
```

Comment:
To check that the loading works as expected we should add more asserts. I suggest to check that the loaded eval_result contains all the values we added to it earlier with `writer.writerow`. Thereby we know that EvalResult loads all the columns as expected and that no values are dropped. The test name `test_load_evaluation_result` is a good fit then.

Reply:
I used a dict comparison instead of adding a lot more asserts. Let me know if this works for you!